src/ripple/app/ledger/impl/LedgerToJson.cpp

```diff
@@ -65,6 +65,10 @@ void fillJson(Object& json, LedgerInfo const& info, bool bFull)
     json[jss::hash] = to_string (info.hash);
     json[jss::totalCoins] = to_string (info.drops);
     json[jss::accepted] = ! info.open;
+    json[jss::close_flags] = info.closeFlags;
+
+    if (info.parentCloseTime)
```

Comment:
The point of this is to allow ledger hash computation so we may need to make more changes so this can be done consistently.